installer/scripts/service_control

```diff
@@ -93,6 +102,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    exit_if_agent_not_onboarded
```

Comment:
The onboarding file in this commit should always register the service after onboarding, unless there is an error during registration itself. But if we ever do get into that situation we could do away with the enable API and merge it with "start" instead. Then "restart" already includes ''start" so it would also register if we are onboarded but not registered.